Sources/QuickTests/FunctionalTests/ItTests.swift

```diff
@@ -14,12 +14,19 @@ class FunctionalTests_ItSpec: QuickSpec {
             expect(exampleMetadata!.example.name).to(equal(""))
         }
 
-        it("has a description with セレクター名に使えない文字が入っている 👊💥") {
-            let name = "has a description with セレクター名に使えない文字が入っている 👊💥"
+        it("has a description with セレクター名に使えない文字が入っている ��") {
+            let name = "has a description with セレクター名に使えない文字が入っている ��"
             expect(exampleMetadata!.example.name).to(equal(name))
         }
 
 #if _runtime(_ObjC)
+        it("is a test with a unique name") {
+            let allSelectors: Set<String> = FunctionalTests_ItSpec.allSelectors();
+            
+            expect(allSelectors).to(contain("is_a_test_with_a_unique_name"));
+            expect(allSelectors).toNot(contain("is_a_test_with_a_unique_name_2"));
+        }
+
```

Comment:
As of now (and still as of the latest version of https://github.com/apple/swift-corelibs-xctest/pull/40), the tests are presented as an array, and the string component is only actually used for display purposes, so a duplicate name shouldn't cause any issues.